packages/material-ui-lab/src/NProgressBar/NProgressBar.js

```diff
@@ -101,7 +101,7 @@ const NProgressBar1 = experimentalStyled(
 
 const NProgressBar = React.forwardRef(function NProgressBar(inProps, ref) {
   const props = useThemeProps({ props: inProps, name: 'MuiNProgressBar' });
-  const { className, initialDelay: defaultInitialDelay, color = 'secondary', ...other } = props;
+  const { className, initialDelay: defaultInitialDelay, color = 'primary', ...other } = props;
```

Comment:
@oliviertassinari I changed the default to `secondary` because people will probably use this together with the **AppBar** component, and to keep the library harmony (without any customization). If we keep `primary`, then he will need to change one of them.

Reply:
Forget it. It doesn't look as bad as I expected. Should have tested before assume that. Thanks!

Reply:
I'm not sure. It feels like the color I have tried is not flying. Not enough contrast. I think that we should reconsider. On Android, Chrome is either blue or white, depending on what contrasts the most. On iOS, Safari uses a blue line. nprogress is blue by default. It looks like a theme.

How about:

- we have a color prop that is null by default, if a user provides a value, it's used. This allows global customizations from the theme
- we have a `bgColor` prop, this is 'primary' by default. It's used to compute the color of the progress, either blue or white.